ci-operator/config/openshift/microshift/openshift-microshift-release-4.21__periodics.yaml

```
@@ -164,6 +164,60 @@ tests:
       MICROSHIFT_OS: rhel-9.6
       SUITE: ai-model-serving
     workflow: openshift-microshift-e2e-bare-metal-tests
+- as: e2e-aws-tests-release-periodic
+  cron: 0 3 * * 0,2,4
+  steps:
+    cluster_profile: aws-2
+    env:
+      EC2_INSTANCE_TYPE: c5.metal
+      MICROSHIFT_OS: rhel-9.6
+      SCENARIO_TYPE: releases
+    workflow: openshift-microshift-e2e-metal-tests
+- as: e2e-aws-tests-release-arm-periodic
```

```suggestion
- as: e2e-aws-tests-release-periodic-arm
```
`arm` must be always at the end